packages/table/src/regions.ts

```diff
@@ -234,6 +236,24 @@ export class Regions {
     }
 
     /**
+     *  Returns the index of the region that wholly contains the supplied
+     * parameter. Returns -1 if no such region is found.
+     */
+    public static findContainingRegion(regions: IRegion[], region: IRegion) {
+        if (regions == null) {
+            return -1;
+        }
+
+        for (let i = 0; i < regions.length; i++) {
+            // containsRegion expects an array of regions as the first param
+            if (Regions.containsRegion([regions[i]], region)) {
```

Comment:
seems kludgy and ripe for refactoring so you can check containment for single regions. possibly a new function that can be reused to check an array?